cmd/copy.go

```diff
@@ -874,21 +889,18 @@ func (cca *cookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 		e := copyDownloadBlobFSEnumerator(jobPartOrder)
 		err = e.enumerate(cca)
 	case common.EFromTo.BlobTrash():
-		e, createErr := newRemoveBlobEnumerator(cca)
-		if createErr != nil {
-			return createErr
-		}
-
-		err = e.enumerate()
+		fallthrough
```

Comment:
We can use commas instead of fallthrough statements, ex:

```go
switch myString {
case "xyz", "abc", "def":
    // do thing
case "ghi":
    // do thing
}```